market-service/src/main/java/rs/edu/raf/banka1/mapper/ForexMapper.java

```diff
@@ -0,0 +1,15 @@
+package rs.edu.raf.banka1.mapper;
+
+import org.springframework.stereotype.Component;
+import rs.edu.raf.banka1.model.Forex;
+
+@Component
+public class ForexMapper {
+    public Forex createForex(String symbols){
+        String[] symbolArr = symbols.split("/");
+        String baseCurrency = symbolArr[0];
+        String quoteCurrency = symbolArr[1];
+        Double exchangeRate = 0.0;
```

Comment:
da li ovo ne moze da se fetchuje?

Reply:
Nije više validno, pošto smo odradili refactor koda sa onim nasljeđivanjem.